rules.bolt

```diff
@@ -0,0 +1,124 @@
+hasRole(role) { auth.token[role] === true }
+
+type CurrentTimestamp extends Number {
+  validate() { this == now }
+}
+
+type InitialTimestamp extends CurrentTimestamp {
+  validate() { prior(this) == null }
+}
+
+type ReadOnlyCoordinator {
+  read() { hasRole("coordinator") }
+}
+
+type WriteOnlyCoordinator {
+  write() { hasRole("coordinator") }
+}
+
+type OnlyNew {
+  validate() { prior(this) == null }
+}
+
+type EditStatus extends WriteOnlyCoordinator {
```

Comment:
I think we can avoid this for now, as various phases will have different statuses. Remove this please.